tests/playwright/src/model/pages/settings-bar.ts

```diff
@@ -44,9 +44,11 @@ export class SettingsBar {
   }
 
   public async openTabPage<T extends SettingsPage>(type: new (page: Page) => T): Promise<T> {
-    const desiredPage = new type(this.page);
-    await (await desiredPage.getTab()).click();
-    return desiredPage;
+    return await test.step(`Open ${type.name} tab from Settings`, async () => {
+      const desiredPage = new type(this.page);
+      await (await desiredPage.getTab()).click();
```

Comment:
I think it may be better to cut the line in two like avoiding 2 await in a row
```
const tab = await desiredPage.getTab();
await tab.click();
```
```suggestion
      const tab = await desiredPage.getTab();
      await tab.click();
```

Reply:
ok, I changed it and split it on two lines even though that was how it was initially.